kge/job/train.py

```diff
@@ -288,9 +312,12 @@ def run_epoch(self) -> Dict[str, Any]:
             epoch=self.epoch,
             split=self.train_split,
             batches=len(self.loader),
-            size=self.num_examples,
-            lr=[group["lr"] for group in self.optimizer.param_groups],
+            num_examples=self.num_examples,
```

Comment:
Since we use size everywhere else (e.g., for the batch size), I suggest to continue using "size" here.